deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -344,6 +344,9 @@ def get(self, request, name, stage=None):
             basic_cluster_info = clusters_helper.get_cluster(request, env.get('clusterName'))
             capacity_info['cluster'] = basic_cluster_info
 
+        console_project_exist = environs_helper.get_nimbus_project(name)
```

Comment:
the variable "existing_stage_identifier" above should already has the console_project info. please use that instead of calling the nimbus API again for better performance.

Reply:
also, the "name" variable here is the env name, it is not the project name, so calling the nimbus api with the env name is not correct. please look at existing_stage_identifier.